src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.HashMap;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String SUPPLY_PRODUCT = "supply";
+    private static final String BUY_PRODUCT = "buy";
+    private static final String RESULT = "result";
+    private static final int INDEX_VALUE = 1;
```

Comment:
maybe we can find better name for `INDEX_VALUE` constant, because we use it not like an index,
but more like a shift

Reply:
One of the hardest things for me is variable names